manifests/deis-logger-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
@jchauncey d'oh! Looks like it's simply a matter of unintended extra whitespace.  Just need to push the block back one space:
```
- metadata:
-   name: deis-logger
-   namespace: deis
-   labels:
-     heritage: deis
-   annotations:
-     helm-keep: "true"
-     component.deis.io/version: canary
+metadata:
+  name: deis-logger
+  namespace: deis
+  labels:
+    heritage: deis
+  annotations:
+    helm-keep: "true"
+    component.deis.io/version: canary
```